### PR TITLE
Feature/task 3

### PR DESCRIPTION
**Pull request by kira2023**

### CODE REVIEW

File: app/pages/course-list-page/directives/course-status.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+angular
+    .module('myApp')
+    .directive('courseStatus', function() {
+        return function(_, element, attrs) {
+            var courseDate = moment(attrs.courseStatus);
```
Comment (kuzanatoliy) [inline app/pages/course-list-page/directives/course-status.js]: ??? you haven't to get variables from attributes.
See https://docs.angularjs.org/guide/directive & https://docs.angularjs.org/guide/component

File: app/styles/app.scss [hunk 2]
```diff
@@ -45,3 +46,15 @@ html, body {
 		box-shadow: 0px 0px 6px -1px $color-black;
 	}
 }
+
+[course-status].actual-course {
```
[the remark this answers is not in the document]
Comment (kira2023) [reply app/styles/app.scss]: fix

File: package.json [hunk 3]
```diff
@@ -6,28 +6,32 @@
...
+    "karma": "3.1.1",
+    "karma-chrome-launcher": "2.2.0",
+    "karma-firefox-launcher": "1.1.0",
+    "karma-jasmine": "1.1.2",
+    "node-sass": "4.12.0",
+    "protractor": "5.4.1"
   },
   "scripts": {
-    "start": "http-server"
+    "start": "gulp && http-server"
```
[the remark this answers is not in the document]
Comment (kira2023) [reply package.json]: fix

File: app/styles/app.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (kira2023) [reply app/styles/app.scss]: it is necessary for increasing specificity of the selector and also we will be sure that these CSS styles will be applied only to element with `course-status` directive

File: app/pages/course-list-page/directives/course-status.js (hunk 1 above)
Comment (kira2023) [reply app/pages/course-list-page/directives/course-status.js]: fix

File: app/shared/app-header/app-header.html [hunk 4]
```diff
@@ -9,11 +9,13 @@ <h1 class="logotype-title">
       </h1>
     </div>
     <div class="header-login-wrapper">
-      <span class="header-login-user">kira@email.com</span>
+      <span class="header-login-user">
+          {{$ctrl.activeUser.email}}
```
[the remark this answers is not in the document]
Comment (kira2023) [reply app/shared/app-header/app-header.html]: fix

File: app/pages/course-list-page/directives/course-status.js (hunk 1 above)
Comment (kira2023) [reply app/pages/course-list-page/directives/course-status.js]: fix

**Pull request merged**